pkg/webhooks/namespace/namespace.go

```diff
@@ -151,7 +153,28 @@ func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl
 			return ret
 		}
 		log.Info("Non-admin access attempt to privileged namespace", "request", request.AdmissionRequest)
-		ret = admissionctl.Denied(fmt.Sprintf("Prevented from creating or changing a potentially harmful Namespace. Customer namespaces should NOT match this regular expressions: %s", privilegedNamespace))
+		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed namespaces. Customer workloads should be placed in customer namespaces, not 'default' or any namespace beginning with 'kube', 'openshift', or 'redhat'."))
+		ret.UID = request.AdmissionRequest.UID
+		return ret
+	}
+
+	if badNamespaceRe.Match([]byte(ns.GetName())) {
+		amISREAdmin := false
+		amIClusterAdmin := utils.SliceContains(request.UserInfo.Username, clusterAdminUsers)
+
+		for _, group := range sreAdminGroups {
+			if utils.SliceContains(group, request.UserInfo.Groups) {
+				amISREAdmin = true
+				break
+			}
+		}
+		if amIClusterAdmin || amISREAdmin {
+			ret = admissionctl.Allowed("Cluster and SRE admins may access")
+			ret.UID = request.AdmissionRequest.UID
+			return ret
+		}
+		log.Info("Non-admin access attempt to privileged namespace", "request", request.AdmissionRequest)
```

Comment:
We can probably change this info message to be specific to DNS.

Reply:
good catch, fixed